### DISCUSSION TITLE
Which version is running in cloud?

### DISCUSSION THREAD

**Discussion by resamaraschi** [sequinstream/sequin#237] [Q&A]
first thank you for the very exciting project. Can one tell me which version of sequinstream is running in cloud? 
I am asking this because, I have some issues running the project locally ([Release v0.4.1](https://github.com/sequinstream/sequin/releases/tag/v0.4.1))
but not in cloud. So i am trying to understand what the issue may be by comparing the version.

**Comment by thisisgoldman** [comment]
Hello @resamaraschi - thanks for writing in!

The cloud version of Sequin will always be the latest release (which right now is [v0.5.0](https://github.com/sequinstream/sequin/releases/tag/v0.5.0)).

We've also started a [Discord Server](https://discord.gg/BV8wFXvNtY) to help provide support. So if you are still hitting issues locally, please send us a note there. We'll also be much more responsive to discussions from here on out.
